### DISCUSSION TITLE
Nested map/multimap collections

### DISCUSSION THREAD

**Discussion by sirinath** [11ty/eleventy#4051] [Enhancement Queue]
This is related to https://github.com/11ty/eleventy/issues/955.

Currently, collections are a list.  But it will be good if there are collections which are key-value mapped. This way I can use collections 'en:books' tp lookup other equivalent collections fr:livres, es:libros, de:Bücher. In this case, there is one context which is the language. There will be other cases where there is no context (moon and luna in and all English astronomy blog) or even multiple contexts (e.g. en-uk&en-gb&en-us:books).

The use cases for this would be:

- internationalisation - list collections which are of another language - the collection name in a primary language against the list of languages which has alternate names mapped to the collection name in those languages - this might help be relevant to [this ](https://github.com/11ty/eleventy/issues/420) issue to manage tags and collections
- synonyms and aliases - when tagging posts certain tags may be the same in the same language or across different languages. These mapping can be made.
- Posts by different authors as a collection

**Comment by zachleat** [comment]
> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

> View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!
